pkg/types/ccipocr3/plugin_commit_types.go

```diff
@@ -74,7 +74,7 @@ type CommitReportInfo struct {
 	// RemoteF Max number of faulty RMN nodes; f+1 signers are required to verify a report.
 	RemoteF     uint64            `json:"remoteF"`
 	MerkleRoots []MerkleRootChain `json:"merkleRoots"`
-	TokenPrices []TokenPrice      `json:"tokenPrices"`
+	PriceUpdates
```

Comment:
Isn't this a breaking change? Or there's a minimal effect to it?

Reply:
Pretty sure it's non-breaking because the embedded type would expose `tokenPrices` at the same json path.